metaseq/modules/quant_noise.py

```diff
@@ -0,0 +1,105 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import torch
+import torch.nn as nn
+
+
+def quant_noise(module, p, block_size):
```

Comment:
@suchenzang, how do we plan to use the quantization noise, do we want to optimise the model for inference later on?